code-challenges.test.js

```diff
@@ -14,38 +14,143 @@
+
+//   })
+// })
 
 
+//* RAN TEST (RESULTS BELOW): *//
+
+// ReferenceError: remainderThree is not defined
+
 // b) Create the function that makes the test pass.
 
+//Goal: Takes in a mixed data array and returns an array of only the REMAINDERS of the numbers when divided by 3
+
+//Pseudocode:
+// Create a function expression remainderThree. 
+// Create a variable that invokes a for loop parameters on array (array). 
+  // Use .push method to move integers divisiable by 3 to front of array. 
+  // Use .pop to remove Boolean values and string values 
+//Run the function and return expected output. 
+
+// const remainderThree= (array) => {
+//   for(let i=0; i<array.length; i++) {
+//     if(array[i] === "integer") {
+//       array.push(array[i] % 3)
+//       return array 
+// # originally on line #97: ... === typeOf Boolean
+//     } else if(array[i] === 'true' & 'false')
+//       array.pop(array[i])
+//     return array 
+//   } else {
+//     return array.pop(array[i])
+//   }
+// }
+
+//* RAN TEST (RESULTS BELOW): *//
+// SyntaxError: /Users/learnacademy/Desktop/week-6-assessment-scharlez/code-challenges.test.js: Unexpected token (101:2)
+
+// ** NOTES TO INSTRUCTOR **: That syntax error is frustrating, I have rearranged the lines and nothing has fixed it, not to mention the code I'm not sure about but I wanted to see if I could use for loops to solve it. 
 
 // --------------------3) Create a function that takes in an array of numbers and returns the sum of all the numbers cubed.
 
 // a) Create a test with an expect statement using the variables provided.
 
-const cubeAndSum1 = [2, 3, 4]
-// Expected output: 99
-const cubeAndSum2 = [0, 5, 10]
-// Expected output: 1125
+describe("sumthingCubed", () => {
+  it("takes in an array of numbers and returns the sum of all the numbers cubed", () => {
+    const cubeAndSum1 = [2, 3, 4]
+    // Expected output: 99
+    const cubeAndSum2 = [0, 5, 10]
+    // Expected output: 1125
+      expect(sumthingCubed(cubeAndSum1)).toEqual(99)
+      expect(sumthingCubed(cubeAndSum2)).toEqual(1125)
+
+  })
+})
 
+//* RAN TEST (RESULTS BELOW): *//
+
+// ReferenceError: sumthingCubed is not defined
 
 // b) Create the function that makes the test pass.
+
+//Goal: Takes in an array of numbers and returns the sum of all the numbers cubed
+
+//Pseudocode:
+// Create a function expression (sumthingCubed). 
+// Set parameter as (output)
+// Create variable to invoke/declare Math.pow method to give cube each index of array. 
+// Use .forEach to iterate over function and determine the sum of the given and now cubed integers. 
+// Use .join to convert array to string
+// Return the desire output.
+
+// const sumthingCubed= (output) => {
+//   // # not used: let cubed = Math.pow([i], 3)
+//   let cubed = Math.pow(cubeAndSum1, 3)
+//   output.forEach(cubed => {
+//     sum +- cubed 
+//     return output.join 
+//   })
+// }
+
+//* RAN TEST (RESULTS BELOW): *//
+
+// ReferenceError: i is not defined
+
```

Comment:
Great work on the pseudocode and thinking through the problem!  Good steps.  I think you would benefit from doing more problems, codewars is good.  Or just google beginner js problems with answers.  You understand what to do, it's just execution that needs some work. You can do this!